update_conda_envs.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+from pathlib import Path
+
+subprocess.run([
+    "pyproject2conda",
+    "project",
+    "--overwrite",
+    "force",
+    "--template-python",
+    str(Path('envs', 'py{py_version}-{env}')),
+])
+
+# Hacky way to install the package in editable mode when creating the env.
+for file in Path('envs').glob('*.yaml'):
+    with open(file, 'a', encoding='utf-8') as f:
+        f.write('  - pip\n')
+        f.write('  - pip:\n')
+        f.write('    - -e ..\n')
+
+# Remove ibis-duck from the Python 3.12 env since it's python-duck
```

Comment:
Released over the weekend if you are talking abount python-duckdb

Reply:
Perfect!